### ISSUE TITLE
Capture script elapsed time

### ISSUE THREAD

**Issue by McFateM** [Islandora-Collaboration-Group/ISLE#129]
Suggest wrapping some ‘date’ commands around some (maybe all) of the install scripts like this:

> #!/bin/bash  
> date  # echo the date at start  
>    ...the script contents...
> date  # echo the date at end

Gives the user a quick record of elapsed time to execute the script.  In particular, I think isle_drupal_build_tools/isnstall_isle_ld_site.sh could benefit from this.

OR, better yet, modify the documentation (This enhancement also appears in the ISLE-Documentation issues queue) so that script execution is fronted with a 'time' command, like so...

```
> root@35bac072fbb3:/tmp/isle_drupal_build_tools# time ./install_isle_ld_site.sh
...
Drush script finished! ...exiting

real	7m8.249s
user	2m37.230s
sys	0m46.770s
root@35bac072fbb3:/tmp/isle_drupal_build_tools# 
```

**Comment by bseeger** [comment]
Good idea - which way do folks want to go with this?  And does 'time' slow any of the commands down?  If so, then I'd advocate for this being in the docs, versus the scripts.

**Comment by dwk2** [comment]
Great suggestion Mark! Since there's little appreciable performance hit,
I'd prefer to see time processing output put into the Code, rather than the
Docs. I think all installations would benefit from this, and I'd like to
keep the documentation as minimal and clean as possible.

David Keiser-Clark
